src/createServer.js

```diff
@@ -1,3 +1,69 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { convertToCase } = require('./convertToCase/convertToCase');
+
+const cases = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+
+const createServer = () => {
+  const server = http.createServer((req, res) => {
+    res.setHeader('Content-Type', 'application/json');
+
+    const normalizedURL = new URL(req.url, `http://${req.headers.host}`);
+    const targetCase = normalizedURL.searchParams.get('toCase');
+    const originalText = normalizedURL.pathname.slice(1);
+    const possibleErrors = validate(originalText, targetCase);
+
+    if (possibleErrors.errors.length) {
+      res.statusCode = 400;
```

Comment:
`If something is not correct, you should respond with 400 status, Bad request statusText`
You don't have a `statusText`.
Check the task requirements, please.